### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by Verurteilt** [binarydud/pyres#150]
I saw the last commit was on May, 2015, is the project still active?

**Comment by steenzout** [comment]
_ping_

**Comment by steenzout** [comment]
@Verurteilt I also found this [https://github.com/Pyres/pyres](https://github.com/Pyres/pyres) but it is way out-of-date (Mar 9, 2012).

**Comment by VictorBefective** [comment]
@steenzout  Yeah I think this project is death.

**Comment by binarymatt** [comment]
yes, this project is no longer maintained. I'll update the repository to put it into archive mode
